ingestion/tests/integration/profiler/test_nosql_profiler.py

```diff
@@ -0,0 +1,188 @@
+from testcontainers.mongodb import MongoDbContainer
+
+from metadata.generated.schema.entity.data.table import ColumnProfile
+from metadata.generated.schema.entity.services.connections.metadata.openMetadataConnection import (
+    OpenMetadataConnection,
+)
+from metadata.generated.schema.entity.services.databaseService import DatabaseService
+from metadata.generated.schema.security.client.openMetadataJWTClientConfig import (
+    OpenMetadataJWTClientConfig,
+)
+from metadata.ingestion.ometa.ometa_api import OpenMetadata
+from metadata.utils.helpers import datetime_to_ts
+from metadata.utils.test_utils import accumulate_errors
+from metadata.utils.time_utils import get_end_of_day_timestamp_mill
+from metadata.workflow.metadata import MetadataWorkflow
+from metadata.workflow.profiler import ProfilerWorkflow
+from metadata.workflow.workflow_output_handler import print_status
+
+SERVICE_NAME = Path(__file__).stem
+
+
+def get_ingestion_config(mongo_port: str, mongo_user: str, mongo_pass: str):
+    return {
+        "source": {
+            "type": "mongodb",
+            "serviceName": SERVICE_NAME,
+            "serviceConnection": {
+                "config": {
+                    "type": "MongoDB",
+                    "hostPort": f"localhost:{mongo_port}",
+                    "username": mongo_user,
+                    "password": mongo_pass,
+                }
+            },
+            "sourceConfig": {"config": {"type": "DatabaseMetadata"}},
+        },
+        "sink": {"type": "metadata-rest", "config": {}},
+        "workflowConfig": {
+            "openMetadataServerConfig": {
+                "hostPort": "http://localhost:8585/api",
+                "authProvider": "openmetadata",
+                "securityConfig": {
+                    "jwtToken": "eyJraWQiOiJHYjM4OWEtOWY3Ni1nZGpzLWE5MmotMDI0MmJrOTQzNTYiLCJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJhZG1pbiIsImlzQm90IjpmYWxzZSwiaXNzIjoib3Blbi1tZXRhZGF0YS5vcmciLCJpYXQiOjE2NjM5Mzg0NjIsImVtYWlsIjoiYWRtaW5Ab3Blbm1ldGFkYXRhLm9yZyJ9.tS8um_5DKu7HgzGBzS1VTA5uUjKWOCU0B_j08WXBiEC0mr0zNREkqVfwFDD-d24HlNEbrqioLsBuFRiwIWKc1m_ZlVQbG7P36RUxhuv2vbSp80FKyNM-Tj93FDzq91jsyNmsQhyNv_fNr3TXfzzSPjHt8Go0FMMP66weoKMgW2PbXlhVKwEuXUHyakLLzewm9UMeQaEiRzhiTMU3UkLXcKbYEJJvfNFcLwSl9W8JCO_l0Yj3ud-qt_nQYEZwqW6u5nfdQllN133iikV4fM5QZsMCnm8Rq1mvLR0y9bmJiD7fwM1tmJ791TUWqmKaTnP49U493VanKpUAfzIiOiIbhg"
+                },
+            }
+        },
+    }
+
+
+TEST_DATABASE = "test-database"
+TEST_COLLECTION = "test-collection"
+TEST_DATA = [
+    {
+        "first_name": "John",
+        "last_name": "Doe",
+        "age": 30,
+    },
+    {
+        "first_name": "Jane",
+        "last_name": "Doe",
+        "age": 25,
+    },
+    {
+        "first_name": "John",
+        "last_name": "Smith",
+        "age": 35,
+    },
+]
+
+
+class NoSQLProfiler(TestCase):
+    """datalake profiler E2E test"""
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        server_config = OpenMetadataConnection(
```

Comment:
nit - we can use https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/tests/integration/integration_base.py#L84 to load the admin ometa

Reply:
dont mind if i do